app/com/openseedbox/jobs/admin/WeeklyMaintenanceJob.java

```diff
@@ -0,0 +1,32 @@
+package com.openseedbox.jobs.admin;
+
+import com.openseedbox.jobs.JobName;
+import com.openseedbox.models.JobEvent;
+import play.Logger;
+import play.jobs.Every;
+
+import java.util.Calendar;
+
+@Every("7d")
+@JobName("Weekly Maintenance Scheduler")
+public class WeeklyMaintenanceJob extends ErrorLoggedAdminJob {
+
+    @Override
+    protected Object doGenericJob() throws Exception {
+        new JobEventJob().now();
+        return null;
+    }
+
+    @JobName("JobEvent Maintenance")
+    public class JobEventJob extends LoggedAdminJob {
+        @Override
+        protected Object doGenericJob() throws Exception {
+            Calendar lastWeek = Calendar.getInstance();
+            lastWeek.add(Calendar.DAY_OF_YEAR, -7);
```

Comment:
It would be nice to externalize this to `openseedbox.maintenance.jobevent.weeks`, and force at least one week.

Reply:
- `openseedbox.maintenance.jobevent.older.than.weeks` and
- `openseedbox.maintenance.torrentevent.older.than.months`